ee/clickhouse/sql/trends/top_elements.py

```diff
@@ -6,7 +6,6 @@
     FROM events e
     WHERE
         team_id = %(team_id)s {entity_query} {parsed_date_from} {parsed_date_to} {prop_filters}
-     AND JSONHas(properties, %(key)s)
```

Comment:
Oh, was this what was preventing none results from returning? I think that may have led me to implement the workaround for unioning the events with no property

Reply:
correct!